push-build.sh

```diff
@@ -45,6 +45,8 @@ PROG=${0##*/}
 #+     [--ci]                    - Used when called from Jekins (for ci runs)
 #+     [--bucket=]               - Specify an alternate bucket for pushes
 #+     [--bucket-mirror=]        - Specify a mirror bucket for pushes
+#+     [--dir-suffix=]           - Specify a suffix to add to the upload
```

Comment:
`--bucket-suffix` makes me think it'll be added to `--bucket`, i.e. `--ci --bucket-suffix=-cross` would upload to `gs://kubernetes-release-dev-cross/ci`.

What I want is `gs://kuberenetes-release-dev/ci-cross`.